spring-ws-security/src/main/java/org/springframework/ws/soap/security/wss4j2/Wss4jSecurityInterceptor.java

```diff
@@ -496,6 +496,14 @@ public void setEnableRevocation(boolean enableRevocation) {
 		this.enableRevocation = enableRevocation;
 	}
 
+	/**
+	 * Enables the {@code useSingleCertificate} attribute on WS-Security headers on outgoing messages. Default is
+	 * {@code true}.
+	 */
+	public void set(boolean useSingleCertificate) {
```

Comment:
I would rather name the method `setUseSingleCertificate`

Reply:
I made the additional changes! Thank you for the review!

Reply:
@corneil if you would have the time to re-review it.